imap_processing/cdf/config/imap_idex_l1a_variable_attrs.yaml

```diff
@@ -126,300 +136,616 @@ target_high_attrs:
+  FIELDNAM: AID
+  LABLAXIS: AID
+
+idx__txhdrfswbincopy:
+  <<: *trigger_base
+  CATDESC: Another copy of the filter/bin
+  FIELDNAM: Filter/Bin
+  LABLAXIS: Filter/Bin
+
+idx__txhdrfswmajor:
+  <<: *trigger_base
+  CATDESC: Major version number for FSW
+  FIELDNAM: Major version
+  VALIDMAX: *max_uint8
+  LABLAXIS: Major Version
+
+idx__txhdrfswminor:
+  <<: *trigger_base
+  CATDESC: Minor version number for FSW
+  FIELDNAM: Minor version
+  VALIDMAX: *max_uint8
+  LABLAXIS: Minor Version
+
+idx__txhdrfswpatch:
+  <<: *trigger_base
+  CATDESC: Patch version number for FSW
+  FIELDNAM: Patch version
+  VALIDMAX: *max_uint16
+  LABLAXIS: Patch Version
+
+idx__txhdrfswhvstat:
+  <<: *trigger_base
+  CATDESC: HVPS polarity setting and status
+  FIELDNAM: HVPS polarity setting and status
+  LABLAXIS: HVPS Polarity
+
+idx__txhdrfswmem0:
+  <<: *trigger_base
+  CATDESC: Read from address specified in param table
+  FIELDNAM: Event Number
+  LABLAXIS: Event Number
+
+idx__txhdrfswmem1:
+  <<: *trigger_base
+  CATDESC: Read from address specified in param table
+
+idx__txhdrfswpt0:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get more detailed description
+
+idx__txhdrfswpt1:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get more detailed description
+
+idx__txhdrfswpt2:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get more detailed description
+
+idx__txhdrfswpt3:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt4:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt5:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt6:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt7:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt8:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfswpt9:
+  <<: *trigger_base
+  CATDESC: Value copied from param table # TODO: get a more detailed description
+
+idx__txhdrfpgaver:
+  <<: *trigger_base
+  CATDESC: Version number of the FPGA version as date code
+  FIELDNAM: FPGA version number
+  LABLAXIS: Version
+
+idx__syncsci0pkt:
+  <<: *trigger_base
+  CATDESC: Synchronization marker
+  FIELDNAM: Synchronization marker
+  VALIDMAX: *max_uint16
+  LABLAXIS: Synchronization Marker
 
-detector_current:
+idx__crcsci0pkt:
   <<: *trigger_base
-  VALIDMAX: 4095
-  FIELDNAM: Detector Current
-  CATDESC: Last measurement in raw dN for processor board signal - "Detector Current "
-  LABLAXIS: Current
-  UNITS: dN
\ No newline at end of file
+  CATDESC: CRC CCITT-16
+  FIELDNAM: CRC CCITT-16
+  VALIDMAX: *max_uint16
+  LABLAXIS: CRC CCITT-16
```

Comment:
I think it was good that you brought in CDF attrs into this YAML. We didn't have this YAML when Bryan worked on this. That's why he had to store in dictionary. This is a nice refactor.